### DISCUSSION TITLE
Saving custom Tag and "Save" shortcut.

### DISCUSSION THREAD

**Discussion by thelephantex** [quodlibet/quodlibet#3557] [Ideas]
Hi everybody, 

I am new to this community and kind of new to everything related to programming, I have been using Ex Falso for a while now...
I have searched through lots of topics and I can`t quite find an answer to these questions, I only found some discussion related to adding a "Save" shortcut for the Save button to pass the hovering of the pointer over the button for every action...
I would also be interested to find out if there is a method to add a new tag in the "+Add" menu of Ex Falso. I am using a few tags that are different from the ones provided by default, for example instead of "Organization" I am using "Publisher" so I have to manually enter it every time for each album I'm editing. I would like to have it saved somehow such that the text auto-fills just as it does for the default tags.

Thanks in advance!

**Comment by declension** [comment]
We only do completion for:
 * tag keys - for known supported tags
 * tag values (any values found so far across the library)

Perhaps this could be an enhancement request I guess, though most people probably want to use standard tags (Where these even exist) for compatibility with other things
